app/controllers/events_controller.rb

```diff
@@ -53,6 +53,6 @@ def set_event
 
     # Only allow a trusted parameter "white list" through.
     def event_params
-      params.require(:event).permit(:name, :description, :gamemode, :sport, :teamsport, :playercount, :gamesystem, :deadline, :startdate, :enddate)
+      params.require(:event).permit(:name, :discipline, :description, :game_mode, :deadline, :startdate, :enddate).merge(player_type: Event.types.first)
```

Comment:
What happened to `playercount`? Did you move it somewhere else or is it not necessary anymore?

Reply:
playercount wasn't mentioned in the original issue so I just removed it. It looked like Constantin added it by himself. See issue 19: https://github.com/hpi-swt2/sport-portal/issues/19

Reply:
Are you sure about this? 

> Maximale Anzahl der teilnehmenden Teams -> if nothing is entered there is no limit

I think it's just named differently in our model

Reply:
oh my bad, readding it